sections/featured-product.liquid

```diff
@@ -444,36 +436,36 @@
                 {%- endif -%}
               {%- when 'icon-with-text' -%}
                 {%- liquid
-                  assign heading1Empty = false
-                  assign heading2Empty = false
-                  assign heading3Empty = false
-                  assign textOnlyAllItems = true
+                  assign heading_1_empty = false
+                  assign heading_2_empty = false
+                  assign heading_3_empty = false
+                  assign text_only_all_items = true
 
                   if block.settings.heading_1 == empty
-                    assign heading1Empty = true
+                    assign heading_1_empty = true
                   endif
 
                   if block.settings.heading_2 == empty
-                    assign heading2Empty = true
+                    assign heading_2_empty = true
                   endif
 
                   if block.settings.heading_3 == empty
-                    assign heading3Empty = true
+                    assign heading_3_empty = true
                   endif
 
-                  if heading1Empty == false and block.settings.icon_1 != 'none' or block.settings.image_1 != null
-                    assign textOnlyAllItems = false
-                  elsif heading2Empty == false and block.settings.icon_2 != 'none' or block.settings.image_2 != null
-                    assign textOnlyAllItems = false
-                  elsif heading3Empty == false and block.settings.icon_3 != 'none' or block.settings.image_3 != null
-                    assign textOnlyAllItems = false
+                  if heading_1_empty == false and block.settings.icon_1 != 'none' or block.settings.image_1 != null
+                    assign text_only_all_items = false
+                  elsif heading_2_empty == false and block.settings.icon_2 != 'none' or block.settings.image_2 != null
+                    assign text_only_all_items = false
+                  elsif heading_3_empty == false and block.settings.icon_3 != 'none' or block.settings.image_3 != null
+                    assign text_only_all_items = false
                   endif
```

Comment:
I kinda feel like we could do without a few of those assigns here as they don't really make a lot of difference. But I understand it's not the scope of this PR. 

```suggestion
                  if block.settings.heading_1 != empty and block.settings.icon_1 != 'none' or block.settings.image_1 != null
                    assign text_only_all_items = false
                  elsif block.settings.heading_2 != empty and block.settings.icon_2 != 'none' or block.settings.image_2 != null
                    assign text_only_all_items = false
                  elsif block.settings.heading_3 != empty and block.settings.icon_3 != 'none' or block.settings.image_3 != null
                    assign text_only_all_items = false
                  endif
```